server/message_hooks.go

```diff
@@ -838,3 +850,13 @@ func (p *Plugin) getMentionsData(message, teamID, channelID, chatID string, clie
 
 	return message, mentions
 }
+
+func getExtension(path string) (string, string) {
```

Comment:
This looks too manual, why not use `path/filepath.Base` to get the filename and `path/filepath.Ext` to get the extension, and then remove the extension from the filename? I think that would be ess error prone and way easier to understand.